checkbox-provider-kivu/bin/gpu-load.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
```

Comment:
Maybe this needs to be `#!/usr/bin/python3` instead?

Reply:
I added these two dependencies in the snapcraft.yaml file so they will be bundled with the snap